rest/restWithParameters/api.graphql

```diff
@@ -1,11 +1,10 @@
-# this shows how query arguments are automatically substituted into the REST call
-# The default for method GET is to append the explode the arguments as query parameters
-# You can generate this entire code using 
+# This example shows how query arguments are automatically substituted into the REST call.
+# The default for method GET is to explode the arguments as query parameters
+# You can generate a similar schema using the StepZen CLI command
```

Comment:
I'd remove any mention of import, have the snippet laser focused on s single topic.